src/modality-specific-files/physiological-and-other-continuous-recordings.md

```diff
@@ -24,7 +24,7 @@ Continuous physiological recordings, such as pulse measurement,
 electrocardiogram, respiratory movement measured with a respiration belt,
 gas concentration, eye-tracking, or head-motion parameters estimated
 by the MRI scanner, SHOULD use `_physio.<tsv.gz|json>` pairs.
-The following recordings MAY be encoded with specific data columns
+We recommend to encode the following recordings with specific data columns
```

Comment:
@mszinte I'll reformulate this suggestion but using the RFC-blah-blah language. In other words, I'll rephrase this paragraph along the following lines:

```Encoding the following recordings with their corresponding specific data columns
as described in section [Specific Physiological Signal Types](#specific-physiological-signal-types)
below is RECOMMENDED:
```

The thing is to use the keyword MAY or RECOMMENDED (it feels you prefer RECOMMENDED, so I'll go with that), as the RFC prescribes.